Python/bytecodes.c

```diff
@@ -3945,16 +3957,16 @@ dummy_func(
 
         op(_SET_IP, (--)) {
             TIER_TWO_ONLY
-            frame->prev_instr = ip_offset + oparg;
+            frame->instr_ptr = ip_offset + oparg;
         }
 
         op(_SAVE_CURRENT_IP, (--)) {
             TIER_ONE_ONLY
-            frame->prev_instr = next_instr - 1;
+            assert(frame->next_instr_offset == 0);
+            frame->next_instr_offset = next_instr - frame->instr_ptr;
```

Comment:
This needs a cast to `(uint16_t)` (really to the type of `next_instr_offset`) otherwise we get warnings from MSVC on the generated files.